.expeditor/deploy.pipeline.yml

```diff
@@ -8,10 +8,13 @@ steps:
         - aws/chef-cd
       executor:
         docker:
-          environment:
-          - DOCKER_CONFIG_JSON_CHEFOPS
-          - AWS_SSL_INTERNAL_ARN
-          - ENVIRONMENT
-          - CI
-          - APP
-          - HAB_AUTH_TOKEN
+      secrets:
+        AWS_SSL_ARN:
```

Comment:
This needs to be `AWS_SSL_INTERNAL_ARN` and the field set to `ssl_internal_arn`. Internal is chef.co, external is chef.io (and opscode.com and a whole bunch of others).